### DISCUSSION TITLE
Hide buildServer.json

### DISCUSSION THREAD

**Discussion by EastHell** [wojciech-kulik/xcodebuild.nvim#43] [Tips & Tricks]
Is there a way to hide buildServer.json file? I don't want it in remote repository, also i don't want to add it in .gitignore separately. Most .gitignore files already ignores hidden folders. Can we move buildServer.json into any hidden folder?

**Comment by wojciech-kulik** [comment]
Most likely it's not possible. This comes from `sourcekit-lsp` implementation: https://github.com/apple/sourcekit-lsp/blob/eca09791c243bb9333783903890f4450d3f42cfc/Sources/SKCore/BuildServerBuildSystem.swift#L44

However, you can open a feature requests there :)

I just added it to my `.gitignore`.

**Reply by EastHell** [reply]
Ty for idea, i found https://gist.github.com/subfuzion/db7f57fff2fb6998a16c this article useful. Works like a charm =)

**Reply by wojciech-kulik** [reply]
Cool thing 🎊  I'm moving it to Tips & Tricks, could be useful for others 🔥 Thank you!
